lib/rspecq/queue.rb

```diff
@@ -241,11 +244,37 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    # Returns the threshold of failures that need to take place before the
+    # build is terminated. This configuration is persisted on a per-build basis
+    # on Redis
```

Comment:
The "This configuration ..." part should be removed, since it's just outlining what the code does (and it's a very simple code). Actually, most of the state here is persisted in Redis.

A more useful comment here can be:

> Returns the number of failures that will trigger the build to fail-fast. Returns 0 if this feature is disabled and nil if the Queue is not yet published